configure.ac

```diff
@@ -2128,6 +2128,57 @@ AC_INCLUDES_DEFAULT
 	AC_SUBST(NETMAP_SRC)
 fi
 
+# Check for DPDK support, defualt no.
+AC_ARG_ENABLE([dpdk],
+[AC_HELP_STRING([--enable-dpdk],[enable dpdk support @<:@default=no@:>@])],
+    [enable_dpdk=$enableval],
+    [enable_dpdk=no])
+
+AC_ARG_WITH([dpdk-includes],
+AC_HELP_STRING([--with-dpdk-includes=IDIR],[Endace DPDK include directory, if not DIR/include]),
+[
+	# User wants DPDK support and has specified a header directory, so use the provided value.
+	enable_dpdk=yes
+	dpdk_include_dir=$withval
+],[])
+
+AC_ARG_WITH([dpdk-libraries],
+AC_HELP_STRING([--with-dpdk-libraries=LDIR],[Endace DPDK library directory, if not DIR/lib]),
+[
+	# User wants DPDK support and has specified a library directory, so use the provided value.
+	enable_dpdk=yes
+	dpdk_lib_dir=$withval
+],[])
+
+if test "x$enable_dpdk" != "xno" ; then
+	# If necessary, set default paths for DPDK API headers and libraries.
+	if test -z "$dpdk_root"; then
+		dpdk_root=/usr/local
+	fi
+
+	if test -z "$dpdk_include_dir"; then
+		dpdk_include_dir="$dpdk_root/include/dpdk"
```

Comment:
Why isn't this just `$dpdk_root/include`, with the `#include` statements including `dpdk/{includefile}`?

If it's always the case that the DPDK include directory is `{rootdirectory}/include` and the the DPDK library directory is `{rootdirectory}/lib`, for some value of `{rootdirectory}`, then you can get rid of the `--with-dpdk-includes` and `--with-dpdk-libraries` options, and simply have a `--with-dpdk` option that takes an optional argument that, if present, is `{rootdirectory}`.  That's how the other optional capture device support options, such as `--with-dag`, work.

Reply:
OK. I have updated configure.ac and CMakelists.txt